tools/README.md

```diff
@@ -0,0 +1,61 @@
+# Using prebuilt images with gRPC OSS benchmarks
```

Comment:
Up until this point, we have tried to remain as decoupled from GCP as possible. The image prefix, for example, allows other Docker registries to be used. Our code does not depend on GKE labels, it uses its own labels. This allows it to easily be ported to run on non-GKE Kubernetes clusters.

These scripts are tightly coupled to GCP, in particular, GCR. They also heavily require the gcloud executable from the google-cloud-sdk. We need to make that clear in the documentation. And, if we intend to continue to try to peruse platform agnosticism, we need to document why these scripts are needed. This will allow people to port them to other platforms. 

/cc @gnossen @paulosjca @jtattermusch

Reply:
Right, these scripts using gcloud specific commands, seems like very closely tight to gcr.io. I think it makes scene to have some explanation in here.